src/schema.js

```diff
@@ -5,38 +5,49 @@ const DEFAULT_LABEL = "…"
 const blocks = {
     api: {
         parents: ['server'],
+        description: "Bundles endpoints under a certain `base_path`.",
+        examples: ['api-proxy'],
         labels: [null, DEFAULT_LABEL]
     },
     backend: {
         parents: ['beta_oauth2', 'definitions', 'jwt', 'oauth2', 'oidc', 'proxy', 'request'],
+        description: "Defines the connection to a local/remote backend service.",
+        examples: ['backend-configuration'],
         labelled: (parentBlockName) => {
             return parentBlockName === "definitions"
         }
     },
     basic_auth: {
         parents: ['definitions'],
+        description: "Access control for Basic Authentication.",
         labelled: true
     },
     beta_oauth2: {
```

Comment:
The id for this block in REFERENCE.md is `oauth2-ac-block-beta`, while this links to `beta-oauth2-block`.

Reply:
Yeah, but IMO the docs should simply follow the convention. There's no use in adding exceptions here and there.

Reply:
So, you want to provide explicit ids to the documentation and not rely on automatic id creation from headerline text?

Reply:
I'd prefer having consistent headlines in the reference: 

```md
## `beta_oauth2` block 
```

( → beta-oauth2-block) instead of 

```md
## OAuth2 CC Block
```

( → oauth2-cc-block)